src/background/Node.js

```diff
@@ -29,9 +30,10 @@ class Node {
   queriedCids = new Set();
 
   async initialize({ rendezvousIp, rendezvousPort, wallet }) {
-    const rendezvousProtocol = /^\d+\.\d+\.\d+\.\d+$/.test(rendezvousIp) ? 'ip4' : 'dns4';
-    const rendezvousWsProtocol = `${rendezvousPort}` === '443' ? 'wss' : 'ws';
-    const rendezvousAddress = `/${rendezvousProtocol}/${rendezvousIp}/tcp/${rendezvousPort}/${rendezvousWsProtocol}/p2p-webrtc-star`;
+    // ToDo: remove these lines at cleanup
+    // const rendezvousProtocol = /^\d+\.\d+\.\d+\.\d+$/.test(rendezvousIp) ? 'ip4' : 'dns4';
+    // const rendezvousWsProtocol = `${rendezvousPort}` === '443' ? 'wss' : 'ws';
+    // const rendezvousAddress = `/${rendezvousProtocol}/${rendezvousIp}/tcp/${rendezvousPort}/${rendezvousWsProtocol}/p2p-webrtc-star`;
 
```

Comment:
```suggestion
```

Reply:
@brunolm Apologies, I merged without this change.  I moved this cleanup into issue #35.